src/assets/scripts/math/circle.js

```diff
@@ -64,3 +64,27 @@ export const fix_angle = (radians) => {
 
     return radians;
 };
+
+/**
+ * normalize angles to within 0° - 360°
+ * @param  {number} degrees an angle
+ * @return {number}         an angle within [0,360]
+ */
+export const degrees_normalize = (degrees) => {
+    if (degrees < 0) {
+        return 360 + (degrees % 360);
+    }
+    else return degrees % 360;
+};
+
+/**
+ * normalize angles to within 0 - 2π
+ * @param  {number} radians an angle
+ * @return {number}         an angle within [0,2π]
+ */
+export const radians_normalize = (radians) => {
+    if (radians < 0) {
+        return (Math.PI*2) + (radians % (Math.PI*2));
+    }
+    else return radians % (Math.PI*2);
```

Comment:
ahh exports.  When I first started using es2015 the whole import/export thing had me stumped more than once.  The syntax was goofy and I didn't understand what the differences were.

```javascript
// default export
// Foo.js
export default class Cube {
   ...
}

// doesn't need curly braces when importing.  Typically default exports are reserved for classes or when a file has a single export.
// bar.js
import Cube from './Foo';

// regular export
// wow.js
export const SOME_CONSTANT = 'A String!';

export const calculateSomething = () => {
    ....
};

// or without the arrow function (which brings in scoping concerns with `this`
export const calculateSomethingElse = function calculateSomethingElse() {
    ....
}

// zoo.js
import { SOME_CONSTANT, calculateSomething, calculateSomethingElse } from './wow';
```

When you are dealing with exported function within the same file, you can use them just as would would normally.
```javascript
export const something = () => {
    ...
};

export const somethingThatDoesSomethingElse = () => {
    const value = something();
}
```

There is one other type, the wildcard import, but that one can have side effects and is inefficient.  The two examples above are what you will see throughout this project.